ravenpy/models/ostrich-blended/raven-blended.rvt.tpl

```diff
@@ -0,0 +1,26 @@
+#########################################################################
+:FileType          rvt ASCII Raven 2.8.2
+:WrittenBy         Juliane Mai & James Craig
+:CreationDate      Sep 2018
+#
+# Generic input file for global models with area averaged series
+#---------------------------------------------------------------
+
+:Gauge meteorological forcings
+   :Latitude    {latitude}
+   :Longitude   {longitude}
+   :Elevation   {elevation}
+   {raincorrection_cmd}
+   {snowcorrection_cmd}
```

Comment:
Could you just replace this with 
```
   :RainCorrection par_x33
   :SnowCorrection par_x34
``` ?
I've tried and test still fail on my end, but maybe I'm missing something else.

Reply:
Yeah, I think it is expected that the test still fails. When I replace it with the above it at least fills in the proper values. Latitude, longitude and elevation are still `None`.

Reply:
I think I know. When calling the model, I've replaced latitude, longitude, elevation arguments by the HRU object, which includes these values and is used to fill the RVH. 
But these are not the same as the latitude, longitude, elevation of the `Gauge` object, which now never are instantiated. 

Up to now, we assumed these two sets of latitude, longitude, elevation were the same thing. This is not a correct assumption, right ?

What is a good solution to specify Gauge latitude, longitude, elevation ? 
- Extraction from the netCDF file (elevation will in most cases not be found)
- User-provided `Gauge` parameters 
- Taking the values from the HRU (status quo)

Reply:
I think in our case it is not really a "gauge" but the basin averaged rain, snow, etc. Then I would provide there also the basin elevation, latitude, and longitude. But it probably won't matter too much.